### PR TITLE
protfolio page with template mods and images

### PR DESCRIPTION
**Pull request by alecorra**
new portfolio page

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <nav>
+        <div class="logo">
+            <img class="mypic" src="img/MyLogo.png">
+        </div>
```
Comment (Liliya-V) [inline portfolio.html]: Why is img wrapped with div, couldn't is be on it's own with margines?

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <a class="home" href="template.html">HOME</a>
+        <a class="about_me" href="about.html">ABOUT ME</a>
+        <a class="portfolio active" href="portfolio.html">PORTFOLIO</a>
+        <a class="contacts" href="#">CONTACTS</a>
+    </nav>
+
+    <div class="content_container_portfolio">
```
Comment (Liliya-V) [inline portfolio.html]: Isn't this class name too massive?

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                <img class="project_link2" src="img/comingsoon1.png"
+            </a>
+        </div>
+
+
+    </div>
+
+    <footer>
+        <div class="links">
+            <a class="git_link" href="https://github.com/alecorra" target="_blank">
```
Comment (Liliya-V) [inline portfolio.html]: .git-link is not used in CSS?

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                <i class="fa fa-github-square"> </i>
+            </a>
+            <a class="linkedin_link" href="https://www.linkedin.com/in/alessandro-corr%C3%A1-ab4a45156/" target="_blank">
```
Comment (Liliya-V) [inline portfolio.html]: .linkedin-link is not used in CSS?

File: css/style.css [hunk 5]
```diff
@@ -139,4 +139,37 @@ nav {
...
+}
+
+.project_link {
+    width: 25%;
+    margin: 2% 10%;
+    border-radius: 30%;
+    border: 2px solid white;
+}
+
+.project_link2 {
```
Comment (Liliya-V) [inline css/style.css]: .project_link and .project_link2 have the only difference: border color. To not to duplicate code you could apply multiple class to the element in HTML

File: css/style.css [hunk 6]
```diff
@@ -139,4 +139,37 @@ nav {
     font-family: "Brush Script MT";
     font-size: 4rem;
     margin-right: 10%;
+}
+
+/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ PORTFOLIO */
+
+.project_container {
+    position: relative;
```
Comment (Liliya-V) [inline css/style.css]: why is set to relative?

File: portfolio.html (hunk 1 above)
Comment (alecorra) [reply portfolio.html]: yes it could, i'll modify later on every page

File: portfolio.html (hunk 3 above)
Comment (alecorra) [reply portfolio.html]: no yet, the class is ready to be controlled in a future

File: css/style.css (hunk 6 above)
Comment (alecorra) [reply css/style.css]: by mistake

File: portfolio.html (hunk 4 above)
Comment (alecorra) [reply portfolio.html]: no yet, the class is ready to be controlled in a future

File: css/style.css (hunk 5 above)
Comment (alecorra) [reply css/style.css]: done

File: css/style.css [hunk 7]
```diff
@@ -139,4 +139,32 @@ nav {
...
+
+/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ PORTFOLIO */
+
+.project_container {
+    width: 90%;
+    text-align: center;
+    margin: 0 auto;
+}
+
+.project_link {
```
Comment (sjr-mayden) [inline css/style.css]: More semantic naming would be project-link-left-col

File: css/style.css (hunk 5 above)
Comment (sjr-mayden) [reply css/style.css]: More semantic naming would be project-link-right-col

File: css/style.css [hunk 8]
```diff
@@ -80,7 +80,7 @@ nav {
     margin-top: 20%;
 }
 
-/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ CONTENT HOME (differnet in every page) */
+/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ HOME (differnet in every page) */
 
 .home_content {
```
Comment (sjr-mayden) [inline css/style.css]: Replace _ with - in class names

File: css/style.css [hunk 9]
```diff
@@ -109,7 +109,7 @@ nav {
     height: 500px;
 }
 
-/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ CONTENT ABOUT ME */
+/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ABOUT ME */
 
 .about_me_pic {
```
Comment (sjr-mayden) [inline css/style.css]: Replace _ with - in class names

File: css/style.css [hunk 10]
```diff
@@ -139,4 +139,32 @@ nav {
     font-family: "Brush Script MT";
     font-size: 4rem;
     margin-right: 10%;
+}
+
+/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ PORTFOLIO */
+
+.project_container {
```
Comment (sjr-mayden) [inline css/style.css]: Replace _ with - in class names

File: portfolio.html (hunk 4 above)
Comment (sjr-mayden) [reply portfolio.html]: Liliya's right, you can always add it back if there is a need for it

File: portfolio.html (hunk 3 above)
Comment (sjr-mayden) [reply portfolio.html]: Liliya's right, you can always add it back if there is a need for it

File: portfolio.html [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+        <div class="project_container">
+            <a class="project1" href="#">
+                <img class="project_link" src="img/comingsoon1.png">
+            </a>
+            <a class="project1" href="#">
+                <img class="project_link project_link2" src="img/comingsoon1.png"
+            </a>
+        </div>
+
```
Comment (sjr-mayden) [inline portfolio.html]: Remove blank lines please

File: portfolio.html [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
```
Comment (sjr-mayden) [inline portfolio.html]: Remove blank lines please

File: template.html [hunk 13]
```diff
@@ -33,7 +33,7 @@
             <a class="linkedin_link" href="https://www.linkedin.com/in/alessandro-corr%C3%A1-ab4a45156/" target="_blank">
                 <i class="fa fa-linkedin-square"> </i>
             </a>
-            <p class="year">Alessandro Corra' 2019</p>
+            <p class="year">Alessandro Corrá 2019</p>
```
Comment (sjr-mayden) [inline template.html]: Rename class something like small-text-footer

**Pull request merged**